app/posts/manage-teacher-training-applications/2020-12-09-improvements-presenting-disability-safeguarding-interview-information.md

```diff
@@ -0,0 +1,136 @@
+
+Similar guidance is also present in other sections of the application details page, and we’d suggest tackling those as a separate piece of work.
+
+Note that there’s currently no facility to store old versions of the guidance, so if it changes on the application side then the content replayed to the provider will not be what the candidate actually saw.
+
+## Showing the safeguarding information
+
+The safeguarding information is hidden to prevent an ‘over the shoulder’ privacy breach.
+
+But there’s a lot of other sensitive personal information within the application which is not hidden in this way. And we don’t think there is much chance of a user using this service in a public place.
+
+That's why we have removed the ability to toggle the display of the safeguarding information. This will mean that users can see all relevant information at a glance.
+
+## Generic wording for users without permission to view safeguarding information
+
+We have new wording for users who cannot see safeguarding information:
+
+> Unavailable: you and your organisation both need permission to view these safeguarding issues.
+
+There’s currently a range of different messages, but we don’t feel that’s necessary now that we have far clearer onboarding and organisational permissions are more likely to be set up.
+
+We think we should instead just provide enough information for users to move to the next step, rather than trying to squeeze in a lot of information.
+
+If the issue of incorrectly set up permissions does persist, we probably need a better overall strategy. One example might be a ‘check my permissions to work out why I can’t see this’ link in appropriate places.
+
+## Distinguishing between system messaging for lack of safeguarding permissions and candidate answers
+
+Currently, the system message that tells the user they can’t view safeguarding permission has the same standard black text as the candidate’s actual answer (when it can be viewed).
+
+We tried using inset text, warning text, details (toggle), a box and grey text when the user lacks permission. But:
+
+- inset text makes it look like a table border
+- warning text is too prominent and is not intended to be used this way
+- the Details component draws attention and requires a click to view a tiny amount of information
+- a box is too prominent and looks messy inside a table
+
+So we decided to use a combination of grey text and a prefix of ‘Unavailable:’.
+
+## Risks
+
+As we’ve worked on this, we’ve discussed the fact that we’re not storing exactly what the user was asked and what answers they could choose from.
+
+Since the candidate service is still being iterated, this means that we risk the chance of misrepresenting a candidate’s answers over time. The answer will remain the same, but the question it appears to be answering will change.
```

Comment:
There should always be a direct relationship between the question being asked and the answer given and this relationship should be immutable. This avoids the question being restated but the answer remaining the same.

Each application would have a set of questions, guidance and answers. Year on year any of these could change. A minor change (ie the premise remains the same) to the question would not require a new question; a major change will.

Reply:
I feel like the difficulty / ambiguity here is when exactly does "the premise remain the same"?

For instance, we’re proposing changing "Start date" to "When did you start this job?" and "End date (leave blank if this is your current role)" to "Are you still working in this job?" and "When did you leave this job?". Are these close enough to be the same, or not?